src/main/utils.js

```diff
@@ -0,0 +1,92 @@
+/*!
+GPII Application
+Copyright 2016 Steven Githens
+Copyright 2016-2017 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+"use strict";
+
+var os       = require("os");
+var fluid    = require("infusion");
+var electron = require("electron");
+
+var app = fluid.registerNamespace("gpii.app");
```

Comment:
Please rewrite this so that the namespace usage is consistent with other files - even though this creates an annoying redundancy:
    var gpii = fluid.registerNamespace("gpii");
    fluid.registerNamespace("gpii.app");

    gpii.app.isWin10OS = function () { ... etc